components/Wallet/EditWalletForm.tsx

```diff
@@ -128,16 +137,41 @@ export default function EditWalletForm ({ wallet, userAddresses, refreshWalletLi
                     </div>
                   </div>
 
-                  <div className={style_pb.btn_row}>
+                  <div className={style_pb.btn_row2}>
                     {error !== '' && <div className={style_pb.error_message}>{error}</div>}
-                    <button type='submit'>Submit</button>
-                    <button onClick={() => { setModal(false); reset() }} className={style_pb.cancel_btn}>Cancel</button>
+                    {wallet.userProfile === null || (wallet.userProfile.isXECDefault === true ?? wallet.userProfile.isBCHDefault === true)
```

Comment:
Think another || here instead of ?? will fix the delete button showing for default bch wallets

Reply:
good catch, solved